src/main/java/org/mule/extension/http/api/request/HttpConnectivityValidator.java

```diff
@@ -130,6 +131,24 @@ private void validateResult(HttpRequest request, Result result) {
     getResponseValidator().validate(result, request);
   }
 
+  //This method is used for composer purposes only
+  public void validate(HttpExtensionClient client, HttpConstants.Protocol protocol, String host, Integer port)
```

Comment:
Why isn't the other validate() method enough?

Reply:
removed it